### PR TITLE
Issue/#1306

### LINKED ISSUE #1306: vagrant windows upgrade fails
unfortunately the .git folder is not copied into the vm
https://github.com/adaptlearning/adapt_authoring/blob/master/vagrant_setup_win.sh#L8
solution make sure hidden files are copied too

### PR DESCRIPTION
**Pull request by lc-thomasberger**
copies .git into VM

Fixes #1306

### CODE REVIEW

**Comment by brian-learningpool** [comment]
+1 (un-tested though)

File: vagrant_setup_win.sh [hunk 1]
```diff
@@ -5,7 +5,10 @@
 cd /vagrant/
 
 # copy the authoring_tool into the VM box
-cp -r * /home/vagrant/
+# this will create the following /home/vagrant/vagrnat
```
Comment (tomgreenfield) [inline vagrant_setup_win.sh]: The tiniest of typos!

**Comment by tomgreenfield** [comment]
+1

**Comment by taylortom** [comment]
+1

**Pull request merged**